src/actions/settings.js

```diff
@@ -106,3 +107,14 @@ export const installAppFirstTime = () => ({
 export const switchCountervalueFirst = () => ({
   type: "SETTINGS_SWITCH_COUNTERVALUE_FIRST",
 });
+
+export const setHideEmptyTokenAccounts = (hideEmptyTokenAccounts: boolean) => (
+  dispatch: *,
+) => {
+  if (setEnvUnsafe("HIDE_EMPTY_TOKEN_ACCOUNTS", hideEmptyTokenAccounts)) {
+    dispatch({
+      type: "SETTINGS_HIDE_EMPTY_TOKEN_ACCOUNTS",
```

Comment:
mmh I think it's an antipattern to have an env both existing in live-common's env and in store. ideally we need to keep state in one place.

if you need something to update based on that env, typically for the checkbox, just use `useEnv` (check with @valpinkman who wrote them. i'm not remembering if they are in live-common yet)

Reply:
>  I think it's an antipattern to have an env both existing in live-common's env and in store

The state of the env has to be saved for it to be restored after the app has been closed. For the desktop implementation I found it more fitting to save it with all the other settings in the redux store rather than follow what has been done for experimentals which are saved in localStorage. I just did it in the same fashion here.

> If you need something to update based on that env, typically for the checkbox, just use `useEnv`

I'm completely at fault on this point, that was pretty stupid of me to rely on the setting state to determine wether to update components while I could have subscribed in a way or another to the env state itself 😅
`useEnv` is not in live-common yet, but `withEnv` can do the trick